decoration.sk

```diff
@@ -530,6 +536,10 @@ directionals before flattening:
 	{skull types} item¦s = minecraft:skull
 	({wall orientations}|{skull floor orientations}) {skull types} = minecraft:skull
 
+	(grass|dirt) path [block]¦s = minecraft:dirt_path
```

Comment:
This will not work because in 1.17, the ID of `dirt path` changed from `grass_path` to `dirt_path` (IIRC)